terraform/main.tf

```diff
@@ -1,7 +1,31 @@
 # Copyright (c) 2023, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
 
+module "network-validation" {
```

Comment:
can you please run terraform fmt in this file and all affected files in this PR?

Reply:
Fixed it.